jvm-libs/linea/metrics/micrometer/src/main/kotlin/net/consensys/linea/metrics/micrometer/MicrometerMetricsFacade.kt

```diff
@@ -8,8 +8,10 @@ import net.consensys.linea.metrics.Histogram
 import net.consensys.linea.metrics.LineaMetricsCategory
 import net.consensys.linea.metrics.MetricsFacade
 import net.consensys.linea.metrics.Tag
+import net.consensys.linea.metrics.TimerCapture
 import java.util.function.Supplier
 import io.micrometer.core.instrument.Counter as MicrometerCounter
+import io.micrometer.core.instrument.Timer as MicrometerTimer
 
 class MicrometerMetricsFacade(private val registry: MeterRegistry, private val metricsPrefix: String) : MetricsFacade {
```

Comment:
I am ware that it's part of your PR, but maybe we should make `metricsPrefix` and optional field and add only when defined.